### PR TITLE
Homework 13 (Natalia Kozak)

### PR DESCRIPTION
**Pull request by Natalia1910**

### CODE REVIEW

File: src/com/company/VowelAndConsonantLetters.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                vowels = vowels + charArray[i]+ " ";
+            }
+            else if (charArray[i]==' ')  { continue;}
+            else consonants = consonants + charArray[i] + " ";
+        }
+        System.out.println(vowels);
+        System.out.println(consonants);
+    }
+
+    public static char[] vowels = new char[]{'а', 'о', 'у', 'е', 'и', 'і'};
```
Comment (ghost-lv) [inline src/com/company/VowelAndConsonantLetters.java]: є ю я ї

File: src/com/company/VowelAndConsonantLetters.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    public static boolean isVowel(char c)
+    {
+        c = Character.toLowerCase(c);
+        for (char d : vowels)
+        {
+            if (c == d)
+                return true;
+        }
+        return false;
```
Comment (ghost-lv) [inline src/com/company/VowelAndConsonantLetters.java]: numbers and symbols and english vowels will be consonants

File: src/com/company/CashMachine.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package com.company;
+
+public class CashMachine {
+
+    public void cash (int money) {
```
Comment (ghost-lv) [inline src/com/company/CashMachine.java]: you must create a method that returns a string

File: src/com/company/VowelAndConsonantLetters.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        String consonants = "";
+
+        for (int i=0;i<charArray.length;i++) {
+            if (isVowel(charArray[i])) {
+                vowels = vowels + charArray[i]+ " ";
+            }
+            else if (charArray[i]==' ')  { continue;}
+            else consonants = consonants + charArray[i] + " ";
+        }
+        System.out.println(vowels);
```
Comment (ghost-lv) [inline src/com/company/VowelAndConsonantLetters.java]: use String as result of this method

File: src/com/company/Main.java [hunk 5]
```diff
@@ -3,11 +3,13 @@
 public class Main {
 
     public static void main(String[] args) {
-	Bottle object = new Bottle();
-		System.out.println(object.t = "Pepsi");
-		System.out.println(object.t = "Cola");
-		System.out.println(object.t = "Fanta");
-		System.out.println(object.t = "Sprait");
+	TasksFromLesson object = new TasksFromLesson();
+	new TasksFromLesson().countOfEvenLetters("lololo");
```
Comment (Volodymyr95) [inline src/com/company/Main.java]: Incorrect class name

File: src/com/company/Main.java [hunk 6]
```diff
@@ -3,11 +3,13 @@
 public class Main {
 
     public static void main(String[] args) {
-	Bottle object = new Bottle();
-		System.out.println(object.t = "Pepsi");
-		System.out.println(object.t = "Cola");
-		System.out.println(object.t = "Fanta");
-		System.out.println(object.t = "Sprait");
+	TasksFromLesson object = new TasksFromLesson();
```
Comment (Volodymyr95) [inline src/com/company/Main.java]: Incorrect variable name

**Pull request merged**